.github/workflows/local-stack-smoke.yml

```
@@ -34,10 +34,47 @@ jobs:
+          env_path = Path('.env')
+          lines = []
+          with env_path.open() as fh:
+              for line in fh:
+                  if line.startswith('OPENAI_API_KEY='):
+                      lines.append(f"OPENAI_API_KEY=\"{os.environ['OPENAI_API_KEY']}\"\n")
+                  else:
+                      lines.append(line)
+          env_path.write_text(''.join(lines))
+          PY
+          python - <<'PY' >> "$GITHUB_ENV"
+import os
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 57: could not find expected ':'

(syntax-check)